### ISSUE TITLE
EditorElement and TEditorElement are not initialising their value with synced v-model

### ISSUE THREAD

**Issue by szabizs** [vueform/vueform#14]
### Environment

any

### Reproduction

Add the v-model to the EditorElement or TEditorElement

### Describe the bug

For the given example, the EditorElement is not rendering it's value from the v-model.

```html
<template>
    <Vueform v-model="form" sync>
        <TextElement name="hello_world" label="Label"/>
        <EditorElement
            id="test-id"
            name="test_description"
            description="Demo description"
            :endpoint="async (file, $el) => {
                await uploadFile(file.file)
            }"
            label="Demo label"
            field-name="file"
            :accept="['jpg', 'jpeg', 'png']"
        />
        <ToggleElement name="toggle">
            Receive notifications
        </ToggleElement>
    </Vueform>

</template>

<script setup>

import { ref } from "vue";
import useFilesApiProvider from "@/utils/FilesApiProvider.js";

const { uploadFile } = useFilesApiProvider();
const form = ref({
    hello_world: "Hello World",
    test_description: "<div>Test</div>",
    toggle: true,
});

</script>
```

**Comment by szabizs** [comment]
The issue is with ref, with reactive it is working. (probably you have to check if its ref or reactive, since ref has *.value*)

**Comment by adamberecz** [comment]
28a37d3 solves this as well

**Comment by adamberecz** [comment]
Fixed in `2.5.5`
